Dockerfile

```diff
@@ -22,7 +22,7 @@
 # Stage 1: Build Rosetta
 # Outputs: binary @ /rosetta/rosetta 
 #---------------------------------------------------------------------
-FROM golang:1.13-alpine as builder
+FROM golang:1.15.5-alpine as builder
```

Comment:
We recommend Go 1.14.12 (which also has the security fix).  We have the patches from upstream to support 1.14.x.  We haven't verified whether 1.15.x would be fine or whether more patches from upstream are necessary to support it.